api/src/test/java/org/apache/iceberg/types/TestTypes.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.types;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestTypes {
+
+  @Test
+  public void fromPrimitiveString() {
+    Assert.assertSame(Types.BooleanType.get(), Types.fromPrimitiveString("boolean"));
```

Comment:
can we change all of those to use AssertJ assertions? That would look similar to:
```
    Assertions.assertThat(Types.BooleanType.get()).isSameAs(Types.fromPrimitiveString("boolean"));

....
    Assertions.assertThat(Types.fromPrimitiveString("Fixed[ 3 ]"))
        .isEqualTo(Types.FixedType.ofLength(3));
....

    Assertions.assertThatThrownBy(() -> Types.fromPrimitiveString("Unknown"))
        .isInstanceOf(IllegalArgumentException.class)
        .hasMessageContaining("...");
```

Reply:
done